arbitrum-docs/run-arbitrum-node/data-availability-committees/04-configure-dac.md

```diff
@@ -0,0 +1,111 @@
+
+Each DAC member gets their own set of BLS public and private keys. It's important for every member to create their own new and secure BLS keys. They should do this on their own and make sure these keys are random and only for their use. If you need help generating BLS keys, please refer to our guide on generating keys in the Arbitrum documentation: [Generating BLS Keys](https://docs.arbitrum.io/node-running/how-tos/running-a-daserver#generate-key).
+
+The main blockchain (parent chain) needs to know the names and public keys of all DAC members in order to validate the integrity of data being batched and posted. A 'keyset' is a list of all DAC members' public keys. It also shows how many signatures are needed to approve a Data Availability Certificate. This design lets the chain owner modify the DAC's membership over time, and it lets DAC members change their keys if needed. See [Inside AnyTrust](/how-arbitrum-works/inside-arbitrum-nitro.md#inside-anytrust) for more information.
+
+In the following section, we will provide a detailed guide on the generation of a Keyset corresponding to your individual set of keys, as well as instructions for its subsequent configuration within the chain.
+
+### Batch poster configuration
+
+AnyTrust works with a group of Data Availability Servers, forming a committee that ensures transaction data is accessible. When setting up the Nitro Batch Poster, you need to provide specific information for each committee member. This includes their URL, BLS public key, a unique single-bit identifier (bitmask) for each member, and a parameter known as assumed-honest. As mentioned before, assumed-honest refers to the minimum number of committee members that we trust.
+
+To ensure data is stored properly, a certain number of committee members need to confirm they have the data. This number is calculated as `K = (N + 1) - H`, where _N_ is the total number of committee members and _H_ is the minimum number of members assumed to be honest.
+
+Someone setting up an AnyTrust node would need to first set up the committee of Data Availability Servers, including generating their BLS keys. How to do that is described [in this guide](https://developer.arbitrum.io/das/daserver-instructions#generate-key).
+
+Here is a sample of the `JSON` configuration, taken from a past configuration of Arbitrum Nova, which was used with to the batch poster configuration. Note that due to a quirk in a configuration library we use in Nitro, the `backends` field is an escaped JSON string with `url`, `pubkey`, and `signer-mask` fields. `pubkey` is the base64 encoded BLS public key of the committee member, and `signer-mask` should be a power of 2, starting from 2^0, 2^1, etc.
```

Comment:
Remove all refs to signer-mask

Reply:
Done, your scanning the diff is appreciated